src/Employer/Employer.Web/Orchestrators/Part2/SkillsOrchestrator.cs

```diff
@@ -1,35 +1,35 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
-using Esfa.Recruit.Employer.Web.Configuration;
 using Esfa.Recruit.Employer.Web.Configuration.Routing;
 using Esfa.Recruit.Employer.Web.Extensions;
 using Esfa.Recruit.Employer.Web.RouteModel;
 using Esfa.Recruit.Employer.Web.ViewModels.Part2.Skills;
 using Esfa.Recruit.Vacancies.Client.Application.Validation;
 using Esfa.Recruit.Vacancies.Client.Domain.Entities;
 using Esfa.Recruit.Vacancies.Client.Infrastructure.Client;
+using Esfa.Recruit.Vacancies.Client.Infrastructure.ReferenceData.Entities;
 using Microsoft.Extensions.Logging;
-using Microsoft.Extensions.Options;
 
 namespace Esfa.Recruit.Employer.Web.Orchestrators.Part2
 {
     public class SkillsOrchestrator : EntityValidatingOrchestrator<Vacancy, SkillsEditModel>
     {
         private const VacancyRuleSet ValidationRules = VacancyRuleSet.Skills;
         private readonly IEmployerVacancyClient _client;
-        private readonly SkillsConfiguration _skillsConfig;
+        private readonly CandidateSkills _candidateSkills;
 
-        public SkillsOrchestrator(IEmployerVacancyClient client, IOptions<SkillsConfiguration> skillsConfigOptions, ILogger<SkillsOrchestrator> logger) : base(logger)
+        public SkillsOrchestrator(IEmployerVacancyClient client, ILogger<SkillsOrchestrator> logger) : base(logger)
         {
             _client = client;
-            _skillsConfig = skillsConfigOptions.Value;
-        }
-
+            _candidateSkills = _client.GetCandidateSkillsAsync().Result;
```

Comment:
I don't think this is a good idea. Can you use a Lazy property that way it only gets populated when it is called the first time.

Reply:
Its always used at least once per request so doing this in constructor is OK. I did initially look at trying to lazyload this but it added extra complexity without enough gain to my mind.

Reply:
suggestion: